packages/keystone/schema.spec.js

```diff
@@ -80,7 +80,7 @@ test('registerSchema without preprocessors', () => {
             User: new GQLListSchema('User', {
                 fields: {
                     name: {
-                        type: Text,
+                        type: 'Text',
```

Comment:
it look like not a part of this PR

Reply:
For some unknown reason, two tests in schema.spec.js were failing in CI. When running them locally, there was an error Text is not defined. And after the error in expect.toMatchObject

Reply:
> For some unknown reason, two tests in schema.spec.js were failing in CI. When running them locally, there was an error Text is not defined. And after the error in expect.toMatchObject

it's fixed in https://github.com/open-condo-software/condo/pull/2136
i think you need to remove this changes